modules/jcommunity/lib/Command/ResetPassword.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Jelix\JCommunity\Command;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ResetPassword extends \Jelix\Scripts\ModuleCommandAbstract
+{
+    protected function configure()
+    {
+        $this
+            ->setName('jcommunity:password:reset')
+            ->setDescription(\jLocale::get('jcommunity~password.reset.cmdline.help.description'))
+            ->setHelp(\jLocale::get('jcommunity~password.reset.cmdline.help.description'))
```

Comment:
help.text, not help.description

Reply:
There is no other argument than the login so help and description are the same for me.